viacoin-blockchain-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "value": "viacoinnode"
+    },
+    "adminUsername": {
+      "value": "azureuser"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "installMethod": {
+      "value": "From_Source"
+    },    
+    "vmSize": {
+      "value": "Standard_D1"
+    }
```

Comment:
I could set the dnsLabelPrefix to
    
"dnsLabelPrefix": {
      "value": "GEN-UNIQUE-11"
    },

I think the other parameters are just fine if I look to other blockchain Azure quickstart templates implimentations